stl/inc/memory

```diff
@@ -796,7 +796,7 @@ protected:
 
     template <class _Ty2>
     void _Weakly_construct_avoiding_expired_conversions_from(const _Ptr_base<_Ty2>& _Other) noexcept {
-        // implement weak_ptr's converting ctors
+        // implement weak_ptr's converting copy ctor

```

Comment:
I'm not fond of the phrase "converting copy/move constructor", it suggests that the constructor being described is a copy/move constructor with some additional properties which is definitely not the case. Could we instead refer to these as "lvalue/rvalue converting constructor" or "copy/move converting constructor"?

Reply:
I like "copy/move converting constructor". Globally cleaned up 5 occurrences.